### ISSUE TITLE
File path bug with webpack 3

### ISSUE THREAD

**Issue by vteivans** [nickjj/manifest-revision-webpack-plugin#40]
Hey!

I upgraded to Webpack 3.6.0 and got this error from the plugin:
```
'./node_modules/css-loader!./node_modules/sass-loader/lib/loader.js?{"sourceMap":true}!./froont/_static/wizard/styles/styles.scss'
    at Error (native)
    at Object.fs.lstatSync (fs.js:982:18)
    at ManifestRevisionPlugin.parsedAssets (/home/vagrant/froont/node_modules/manifest-revision-webpack-plugin/index.js:78:16)
    at Compiler.<anonymous> (/home/vagrant/froont/node_modules/manifest-revision-webpack-plugin/index.js:166:33)
    at Compiler.applyPlugins (/home/vagrant/froont/node_modules/webpack/node_modules/tapable/lib/Tapable.js:61:14)
    at Watching._done (/home/vagrant/froont/node_modules/webpack/lib/Compiler.js:104:17)
    at compiler.emitRecords.err (/home/vagrant/froont/node_modules/webpack/lib/Compiler.js:78:19)
    at Compiler.emitRecords (/home/vagrant/froont/node_modules/webpack/lib/Compiler.js:375:38)
    at compiler.emitAssets.err (/home/vagrant/froont/node_modules/webpack/lib/Compiler.js:61:20)
    at applyPluginsAsyncSeries1.err (/home/vagrant/froont/node_modules/webpack/lib/Compiler.js:368:12)
    at next (/home/vagrant/froont/node_modules/webpack/node_modules/tapable/lib/Tapable.js:218:11)
    at Compiler.compiler.plugin (/home/vagrant/froont/node_modules/webpack/lib/performance/SizeLimitsPlugin.js:99:4)
    at Compiler.applyPluginsAsyncSeries1 (/home/vagrant/froont/node_modules/webpack/node_modules/tapable/lib/Tapable.js:222:13)
    at Compiler.afterEmit (/home/vagrant/froont/node_modules/webpack/lib/Compiler.js:365:9)
    at require.forEach.err (/home/vagrant/froont/node_modules/webpack/lib/Compiler.js:354:15)
    at /home/vagrant/froont/node_modules/webpack/node_modules/async/dist/async.js:421:16
```

I have no idea what caused it since I don't know exactly how Webpack works and how it compiles things, but I was able to fix the issue by reordering items in the `if` statement where the error occurred:
**Original:**
```js
        // Attempt to ignore chunked assets and other unimportant assets.
        if (item.name.indexOf('multi ') === -1 &&
            item.name.indexOf('~/') === -1 &&
            item.reasons.length === 0 &&
            fs.lstatSync(item.name).isFile() &&    // <-- fails here
            item.hasOwnProperty('assets') &&
            item.assets.length === 1) {

            addCurrentItem = true;
        }
```

**Adjusted:**
```js
        // Attempt to ignore chunked assets and other unimportant assets.
        if (item.name.indexOf('multi ') === -1 &&
            item.name.indexOf('~/') === -1 &&
            item.reasons.length === 0 &&
            item.hasOwnProperty('assets') &&
            item.assets.length === 1 &&
            fs.lstatSync(item.name).isFile()) {

            addCurrentItem = true;
        }
```

There could be something off with my configuration, but for the plugin, it has stayed exactly the same as it was in the first Webpack.

**Comment by vteivans** [comment]
After a little bit of searching, I did find that the code is updated. But it seems not to be published to npm. Can it be fixed?

**Comment by nickjj** [comment]
Thanks for the reminder. I just published v0.4.1 which has this fix in.

**Comment by vteivans** [comment]
Thanks a lot 👍
